assets/js/persistence.js

```diff
@@ -109,6 +115,7 @@ function persistChoices(mode, updatePermalink) {
 
         var qName = '';
         if(hash === '#translation') { qName = ''; }
+        if(hash === '#webpageTranslation') { qName = 'P'; }
         if(hash === '#analyzation') { qName = 'A'; }
         if(hash === '#generation') { qName = 'G'; }
```

Comment:
Are you suggesting something like the following?

```
var hashURLMap = {
    '#translation': ' ',
    '#webpageTranslation': 'P',
    '#analyzation': 'A',
    '#generation': 'G'
};
```

and then I access the values by :  `hashURLMap.hash` ?